.github/workflows/release-docs.yml

```diff
@@ -123,6 +145,15 @@ jobs:
           asset_path: ./asgardeo-docs-${{ env.NEW_VERSION }}.zip
           asset_name: asgardeo-docs-${{ env.NEW_VERSION }}.zip
           asset_content_type: application/zip
+      
+      - name: Upload Prod Release Asset
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: ./asgardeo-docs-${{ env.NEW_VERSION }}-prod.zip
+          asset_name: asgardeo-docs-${{ env.NEW_VERSION }}-prod.zip
+          asset_content_type: application/zip
+
```

Comment:
remove unwanted newline

Reply:
fixed with https://github.com/wso2/docs-is/pull/4906/commits/e1eb25cb2cb61a04b21ed5d612fdcf02135098ef